lib/panko/serializer.rb

```diff
@@ -75,7 +75,7 @@ def method_added(method)
 
       def has_one(name, options = {})
         serializer_const = options[:serializer]
-        serializer_const = Panko::SerializerResolver.resolve(name.to_s, self) if serializer_const.nil?
+        serializer_const = Panko::SerializerResolver.resolve(name.to_s, self) if serializer_const.nil? || serializer_const.kind_of?(String)
```

Comment:
@yosiat Is there a reason why you added support in the `has_one` method, but not the `has_many` method?

Reply:
@ryoung umm don't think there was a specific reason..

want to submit a PR for that?

Reply:
Yep, I can do that.